### DISCUSSION TITLE
Spotify downloads

### DISCUSSION THREAD

**Discussion by djd711** [Shabinder/SpotiFlyer#1172] [Q&A]
When I use a Spotify link, does Spotiflyer actually download it from Spotify? I have my Spotify set to very high streaming and download quality which should be 320k however when I download a track with a Spotify URL, the bit rate only shows 128k.

If it's not actually using Spotify, I take it Spotiflyer is falling back to another platform which is fine but should it really be advertised as using Spotify if it doesn't?

Or am I simply missing something 🤦

Thanks.

**Comment by Kendu26** [comment]
Yes, Spotiflyer downloads from Spotify ... If you want to download the 320k of any track, change the preferred audio quality through settings in the Spotiflyer app. I think there should be about 5 different qualities available ranging from 128kbps - 320kbps.

**Reply by Tim-Ceho** [reply]
I downloaded all of my songs with 320kbps prefered but they still downloaded in 128kbps

**Comment by MembershipExcel** [comment]
Other thread 
[https://github.com/Shabinder/SpotiFlyer/discussions/786]()

**Comment by ksaciec** [comment]
Besides, you can [get music files from Spotify with Musify Music Downloader](https://echoshare.co/put-music-on-mp3-player-from-spotify/), which is also compatible with YouTube, Facebook, and other popular websites.
Click the link, and you will get the instructions in detail.

**Comment by kistio83** [comment]
Anche a me succede stessa cosa, sia da PC che da cellulare ANDROID.
Qualsiasi qualità audio metto scarica sempre a 128kbps.
Grazie
